localstack/services/apigateway/provider.py

```diff
@@ -241,6 +241,8 @@ def update_rest_api(
 
     @handler("PutRestApi", expand=False)
     def put_rest_api(self, context: RequestContext, request: PutRestApiRequest) -> RestApi:
+        # TODO: take into account the mode: overwrite or merge
```

Comment:
yep, no "mode" means "merge" - not necessarily a fix for now, but we could return a non-supported error if the mode is overwrite

Reply:
Right, but right now we are not merging but always overwriting I believe?